checkov/secrets/runner.py

```diff
@@ -127,7 +133,13 @@ def run(
                     filter_ignored_paths(root, d_names, excluded_paths)
                     filter_ignored_paths(root, f_names, excluded_paths)
                     for file in f_names:
-                        if file not in PROHIBITED_FILES and f".{file.split('.')[-1]}" in SUPPORTED_FILE_EXTENSIONS:
+                        if runner_filter.secret_scan_extension_by_file_type:
```

Comment:
I would recommend to change the condition logic here in general. Instead of `if ... elif ... elif` -> `if ... if ... if` then you don't have to repeat the `f".{file.split('.')[-1]}" in SUPPORTED_FILE_EXTENSIONS:` part each time